test/unit/event.js

```diff
@@ -2686,6 +2686,134 @@ test( "Inline event result is returned (#13993)", function() {
 	equal( result, 42, "inline handler returned value" );
 });
 
+test( "preventDefault() on focusin does not throw exception", function( assert ) {
+	expect( 1 );
+
+	var done = assert.async(),
+		input = jQuery( "<input/>" ).appendTo( "#form" );
+		input
+			.on( "focusin", function( event ) {
+				var exceptionCaught;
+
+				try {
+					event.preventDefault();
+				} catch ( theException ) {
+					exceptionCaught = theException;
+				}
+
+				assert.strictEqual( exceptionCaught, undefined,
+					"Preventing default on focusin throws no exception" );
+
+				done();
+			} )
+			.focus();
+} );
```

Comment:
I still dislike the whole jQuery style guide, but this space looks really wrong IMHO. Though it's consistent so I don't think we should do exceptions (like PEP did for example).